app/views/users/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>This is a view to show a single user</h1>
\ No newline at end of file
+<%= render partial: "partials/user", locals: {user: @user} %>
+
+<section class="bio-section">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</section>
+
+<section class="posts-section">
+  <h2>Posts</h2>
+  <div class="posts-container">
+    <ul class="posts-list">
+    <% @posts.each do |post| %>
+      <%= render partial: "partials/post", locals: {post: post} %>
+    <% end %>
+    </ul>
+  </div>
+  <%= link_to user_posts_path(@user.id) do %>
+    <button>Show all posts</button>
+  <% end %>
```

Comment:
- [ ] As requested by the previous reviewer the User should see all posts while clicking on the `show all posts` button on the user's show page. Currently, I am getting the following syntax error, could you fix this?
![image](https://user-images.githubusercontent.com/81932478/214422964-82273c27-45f9-4ce9-ab13-c5a0a86e6740.png)

Reply:
Hi @Munsa1 :wave: 
Could you please tell me what actions you took prior to getting this error ? It is working on my end, and i'm having a hard time figuring out how this happened